packages/ibm-products/src/components/Datagrid/Datagrid.test.js

```diff
@@ -1863,67 +1866,107 @@ describe(componentName, () => {
     );
   });
 
-  const RightAlignedColumns = () => {
-    const columns = React.useMemo(
-      () => [
-        ...defaultHeader.slice(0, 3),
-        {
-          Header: 'Age',
-          accessor: 'age',
-          rightAlignedColumn: true,
-        },
-        {
-          Header: 'Visits',
-          accessor: 'visits',
-          rightAlignedColumn: true,
-        },
-      ],
-      []
-    );
+  const rightAlignedColumnsData = [
+    ...defaultHeader.slice(0, 3),
+    {
+      Header: 'Age',
+      accessor: 'age',
+      rightAlignedColumn: true,
+      disableSortBy: true,
+    },
+    {
+      Header: 'Visits',
+      accessor: 'visits',
+      rightAlignedColumn: true,
+    },
+  ];
+
+  const centerAlignedColumnsData = [
+    ...defaultHeader.slice(0, 3),
+    {
+      Header: 'Age',
+      accessor: 'age',
+      centerAlignedColumn: true,
+      disableSortBy: true,
+    },
+    {
+      Header: 'Visits',
+      accessor: 'visits',
+      centerAlignedColumn: true,
+    },
+  ];
+
+  const CustomAlignColumns = ({ customCols }) => {
+    const columns = React.useMemo(() => customCols, [customCols]);
```

Comment:
If `customCols` is a JS object, then assigning it directly to a variable is the fastest way.

I.e. replace this

~`const columns = React.useMemo(() => customCols, [customCols]);`~

with this
```
const datagridState = useDatagrid(
  {
    columns: customCols,
    data,
  }
```